sdk/keyvault/keyvault-admin/swagger/README.md

```diff
@@ -0,0 +1,19 @@
+# KeyVault Admin Swagger Configuration
+
+> see https://aka.ms/autorest
+
+``` yaml
+typescript:
+  package-name: "@azure/keyvault-admin"
+use-extension:
+  "@microsoft.azure/autorest.typescript": "~5.0.1"
+azure-arm: false
+generate-metadata: false
+add-credentials: true
```

Comment:
In #9635, we are setting `add-credentials` to `false`. Should we do the same here?